x-pack/docs/en/security/limitations.asciidoc

```diff
@@ -51,7 +52,7 @@ When a user's role enables document or field level security for a data stream or
 ** Update requests included in bulk requests aren't supported.
 * The request cache is disabled for search requests.
 
-When a user's role enables document level security for a data stream or index:
+When a user's role enables <<document-level-security,document level security>> for a data stream or index:
 
 * Document level security isn't applied for APIs that aren't document based.
   An example is the field stats API.
```

Comment:
@ywangd, the [field stats API](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/search-field-stats.html) was deprecated in 5.4.0. Should we instead list the [field capabilities API](https://www.elastic.co/guide/en/elasticsearch/reference/7.9/search-field-caps.html) as an example?

Reply:
This is a good catch. I am actually not entirely sure what this statement means. If it means field stats API does not respect document level security (DLS) and could leak field information, I don't know whether we still have another example similar to this. For example, the field capabilities API does respect DLS. Given DLS/FLS is implemented at a level close to Lucene, I doubt we have another example similar to the now-removed field stats API (if there is, it should be a bug). So I think we might want to remove this sentence. @tvernum Could you please comment on this in case I am missing something? Thanks!

Reply:
Agreed @ywangd that this statement seems outdated and should probably be removed. Will wait for @tvernum to respond.

Reply:
I removed the statement referencing the stats API. It just didn't seem to make sense anymore since it was deprecated, and we don't seem to have another API that doesn't respect DSL.